README.md

```diff
@@ -105,6 +105,33 @@ Aditionally, the import snippet will replace your clipboard content:
 ```jsx
 import MyTestComponent from '../components/MyTestComponent.tsx';
 ```
+### Copy Partytown files script
+
+The partytown library need the web workers and service workers static files to work, this script copy those required files. More info: https://partytown.builder.io/copy-library-files
+Add the `copyPartytown` task to your `deno.json` file:
```

Comment:
Suggestion:

Move these instructions on how to add a script into a single section above, already with all the tasks that `$live` exports. Does that make sense, since now we're exporting more than one?

Also, it would make this doc only about what the script does.

Just a nitpick